packages/khalti/pubspec.yaml

```diff
@@ -1,21 +1,25 @@
 name: khalti
-description: An official Flutter plugin for Khalti Payment Gateway, to accept payments using Khalti Wallet, E-banking, Mobile Banking, Connect IPS and SCT cards.
+description: An official Flutter plugin for Khalti Payment Gateway, to accept
+  payments using Khalti Wallet, E-banking, Mobile Banking, Connect IPS and SCT
+  cards.
 version: 1.0.2
 homepage: https://khalti.com/payment-gateway/
 documentation: https://docs.khalti.com/
 issue_tracker: https://github.com/khalti/khalti-flutter-sdk/issues
 repository: https://github.com/khalti/khalti-flutter-sdk/tree/master/packages/khalti
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.17.6 <3.0.0"
```

Comment:
Is there any specific reason to upgrade this? This will be a breaking change for those who're using the package in older verison of Dart or Flutter Version < 3.

Reply:
i think I was not able to use `typedef` with the older version

Reply:
@sarbagyastha In order to write
```dart
typedef InnerClient = http.Client;
```
This requires the `nonfunction-type-aliases` language feature to be enabled for which the minimum sdk constraint is `2.17.0`.

Reply:
Let's set the min version to `2.17.0` then.